libs/utils/src/lib/emails/utils.ts

```diff
@@ -178,12 +187,40 @@ export function getEventEmailData({ event, orgName, attachment = true, email, in
     id: event.id,
     title: event.title,
     start: format(eventStartDate, 'Pppp'),
+    otherTimezones: {
+      centralAmerica: getTimezoneEventStartHour(eventStartDate, "America/Chicago"),
+      easternAmerica: getTimezoneEventStartHour(eventStartDate, "America/New_York"),
+      centralEurope: getTimezoneEventStartHour(eventStartDate, "Europe/Helsinki"),
+      westernEurope: getTimezoneEventStartHour(eventStartDate, "Europe/Paris"),
+      japan: getTimezoneEventStartHour(eventStartDate, "Asia/Tokyo"),
+      pacific: getTimezoneEventStartHour(eventStartDate, "America/Vancouver"),
+    },
     end: format(eventEndDate, 'Pppp'),
     type: eventTypes[event.type],
     viewUrl: `/event/${event.id}/r/i${eventUrlParams}`,
     sessionUrl: `/event/${event.id}/r/i/session${eventUrlParams}`,
     accessibility: event.accessibility,
-    calendar: attachment ? getEventEmailAttachment(event, orgName) : undefined
+    calendar: attachment ? getEventEmailAttachment(event, orgName) : undefined,
+    eventDuration: getEventDuration(eventStartDate, eventEndDate)
+  }
+}
+
+function getTimezoneEventStartHour(date: Date, timezone: string) {
+  const tzDate = new Date(date.toLocaleString("en-us", {timeZone: timezone}));
+  return format(tzDate, 'h:mm a');
+}
+
+function getEventDuration(start: Date, end: Date): string {
+  const startTs = start.getTime();
+  const endTs = end.getTime();
+  const duration = intervalToDuration({start, end});
+  
+  if (endTs - startTs < millisecondsInHour) {
+    return formatDuration(duration, {format: ['minutes']});
+  } else if (endTs - startTs < millisecondsInHour * 72) {
+    return formatDuration(duration, {format: ['hours']});
+  } else {
+    return `${differenceInDays(endTs, startTs)} days`;
```

Comment:
```suggestion
  const duration = intervalToDuration({start, end});
  
  if (duration.hours < 2) {
    return formatDuration(duration, {format: ['minutes']});
  } else if (duration.days < 3) {
    return formatDuration(duration, {format: ['hours']});
  } else {
    return `${differenceInDays(endTs, startTs)} days`;
  }
```

It's not fully the same logic but I think it's better. With your function, if the duration would be 1 hour and 5 minutes, it would return "1 hours"